lib/javascript/manage-food-table.js

```diff
@@ -1,23 +1,53 @@
-// I should see a table of all my foods, with Name, Calories and a delete icon for each food
-// <tr>
-// <td>This is a name.</td>
-// <td>This is a calorie.</td>
-// <td>This is a delete icon.</td>
-// <tr> -->
-// tableclass is manage-foods
-
-// https://quantify-this-api.herokuapp.com/api/v1/foods
-// return $.getJSON(`${host}/posts`)
 $ = require('jquery');
 
 $(document).ready(function(){
   getAllFoods();
+  $('.new-food-form').on('submit', function(){
+    event.preventDefault();
+    var foodName = $('input[name="new-food-name"]').val();
+    var foodCalories = $('input[name="new-food-calories"]').val();
+    postNewFood({food: {name: foodName, calories: foodCalories}});
+    $('.manage-foods > tbody > tr:first').after(`<tr><td>${foodName}</td><td>${foodCalories}</td><td><div class="delete-icon"></div></td><tr>`);
```

Comment:
you may want this line within the `postNewFood` function. we don't want to append the new food to our page unless it was successfully saved in our database, right?